featuros-plugin/build.gradle

```diff
@@ -1,2 +1,9 @@
 project(':featuros-plugin') {
+    dependencies {
+        compile 'org.jodd:jodd-http:3.6.6'
```

Comment:
@tadaskay, I haven't heard about `Jodd` before, but it looks very interesting - much more simpler and readable than Groovy's httpBuilder. Would you recommend it for the usage in other projects?